src/irr/asset/CImageWriterJPG.cpp

```diff
@@ -11,14 +11,12 @@
 #include "irr/asset/ICPUImageView.h"
 #include "os.h"
 
-#include "os.h"
-
 #ifdef _IRR_COMPILE_WITH_LIBJPEG_
 #include <stdio.h> // required for jpeglib.h
 extern "C"
 {
-	#include "libjpeg/jpeglib.h"
-	#include "libjpeg/jerror.h"
```

Comment:
this is how I like my include paths, so I can see the 3rdparty lib they come from

Reply:
you mean?

Reply:
I did it because my project could not see those files

Reply:
maybe we should change it in cmake

Reply:
`./3rdparty` is an include path already, you should be seeing it

Reply:
but it did not work for me

Reply:
when building or Intellisense?

Reply:
when building

Reply:
works for me and criss it seems on our branches